setup.cfg

```diff
@@ -52,3 +52,14 @@ filterwarnings =
      default::DeprecationWarning:tests\.(.*)|parsec\.(.*)
      # Ignore our todo warnings
      ignore:TODO:UserWarning:parsec|tests
+markers =
+    linux: marks tests as linux only
+    win32: marks tests as Windows only
+    gui: marks tests as GUI (select with '--rungui')
+    slow: marks tests as slow (select with '--runslow')
+    mountpoint: marks tests as mountpoint (select with '--runmountpoint')
+    postgresql
```

Comment:
"Select with `--postgresql`" ?

Reply:
So the `@pytest.mark.postgresql` tests are skipped when `--postgresql` is not provided?

Reply:
yup (on top of having backend_factory using the postgresql implementation instead of the memory mock one)